src/ontology/mondo-ingest.Makefile

```diff
@@ -365,7 +365,12 @@ slurp/%.tsv: $(COMPONENTSDIR)/%.owl $(TMPDIR)/mondo.sssom.tsv $(REPORTDIR)/%_ter
 	--slurp-dir-path slurp/ \
 	--outpath $@
 
-slurp-%:
+slurp-no-updates-%:
```

Comment:
New slurp make goals to run the pipeline w/out updating prereqs. Useful if we know that the prereqs are up to date and want to run it faster.
```make
slurp-no-updates-%:
	$(MAKE) slurp/$*.tsv

slurp-all-no-updates: slurp-no-updates-omim slurp-no-updates-doid slurp-no-updates-ordo slurp-no-updates-icd10cm slurp-no-updates-icd10who slurp-no-updates-ncit
```

Reply:
Good idea.